config/entityProviders.json

```diff
@@ -6,7 +6,7 @@
   "organizationsettings": "env://ENTITY_PROVIDER_ORGANIZATION_SETTINGS?default=firstconfigured",
   "organizationmembercache": "env://ENTITY_PROVIDER_ORGANIZATION_MEMBER_CACHE?default=firstconfigured",
   "repository": "env://ENTITY_PROVIDER_REPOSITORY?default=firstconfigured",
-  "repositorycache": "env://ENTITY_PROVIDER_REPOSITORY_CACE?default=firstconfigured",
+  "repositorycache": "env://ENTITY_PROVIDER_REPOSITORY_CACHE?default=firstconfigured",
```

Comment:
Maybe a deployment issue to create a new cache

Reply:
Oh that's embarrassing! Yeah, a deployment issue if anyone has over-ridden that entity.

Might be time for a `changelog.md` to help track version changes of note and at least increment the minor version of the package or something like that...